nmostesting/mocks/Node.py

```diff
@@ -572,33 +573,41 @@ def transport_file(version, resource, resource_id):
     # GET should either redirect to the location of the transport file or return it directly
     try:
         if resource == 'senders':
-            template_path = "test_data/controller/video.sdp"
+            sender = NODE.senders[resource_id]
+            sdp_params = {**CONFIG.SDP_PREFERENCES, **sender.get('sdp_params', {})}
+
+            sdp_file = "jxsv.sdp" if sdp_params.get("media_type", "raw") == "jxsv" else "video.sdp"
+
+            template_path = "test_data/controller/" + sdp_file
 
             template_file = open(template_path).read()
             template = Template(template_file, keep_trailing_newline=True)
 
-            sender = NODE.senders[resource_id]
             destination_ip = sender['activations']['transport_params'][0]['destination_ip']
             destination_port = sender['activations']['transport_params'][0]['destination_port']
             source_ip = sender['activations']['transport_params'][0]['source_ip']
 
-            interlace = ""
-            if CONFIG.SDP_PREFERENCES["video_interlace"] is True:
-                interlace = "interlace; "
-            # TODO: The SDP_PREFERENCES doesn't include video media type
+            # TODO: The SDP_PREFERENCES doesn't include video media type, or ST2110 21 Sender Type
```

Comment:
Isn't `st2110_21_sender_type` the same thing as `type_parameter`?

Reply:
As usual, sir, you are absolutely correct :) - corrected in db8c1fb